extension/main.js

```diff
@@ -86,7 +86,11 @@
 			ghTab.url = url + 'notifications';
 		}
 
-		if (tab.url === '' || tab.url === 'chrome://newtab/' || tab.url === ghTab.url) {
+		if (window.GitHubNotify.settings.get('useParticipatingCount')) {
+			ghTab.url += '/participating';
+		}
+
+		if (typeof tab !== 'undefined' && (tab.url === '' || tab.url === 'chrome://newtab/' || tab.url === ghTab.url)) {
```

Comment:
I guess you can leave it, but the docs say a tab is to be returned, so that's either a doc bug or behaviour bug. https://developer.chrome.com/extensions/browserAction#event-onClicked